src/main/java/fi/aalto/cs/apluscourses/utils/observable/ObservableProperty.java

```diff
@@ -80,6 +80,14 @@ public T getAndSet(T value) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Notifies all observers with the current value. Works similarly to {@link
+   * ObservableProperty#set}, but the actual value isn't changed.
+   */
+  public void valueChanged() {
+    onValueChanged(get(), null);
+  }
```

Comment:
@OlliKiljunen was way ahead of me here :smile: 
https://github.com/Aalto-LeTech/intellij-plugin/pull/211/files/fcd4f1fb452a58e3f81ec4865a7fefa32207c4b6#r437382799